tck/src/main/java/org/eclipse/microprofile/openapi/tck/ModelConstructionTest.java

```diff
@@ -456,6 +456,20 @@ public void mediaTypeTest() {
     public void schemaTest() {
         final Schema s = processConstructible(Schema.class);
         
+        final Schema ap = createConstructibleInstance(Schema.class);
+        checkSameObject(s, s.additionalPropertiesSchema(ap));
+        checkSameObject(ap, s.getAdditionalPropertiesSchema());
+        assertEquals(s.getAdditionalPropertiesBoolean(), null, "AdditionalProperties (Boolean type) is expected to be null");
+        checkSameObject(s, s.additionalPropertiesBoolean(Boolean.TRUE));
+        assertEquals(s.getAdditionalPropertiesBoolean(), Boolean.TRUE, "AdditionalProperties (Boolean type) is expected to be true");
+        assertEquals(s.getAdditionalPropertiesSchema(), null, "AdditionalProperties (Schema type) is expected to be null");
```

Comment:
I am trying to understand why the schema property would be `null` here, and not `ap`.  Is it because we set the boolean property, and therefore would have automatically set the schema property to null?  If so we should change the javadoc of the methods to specify this behaviour, and also change the default interface impl to set the other variant to null.